File: PrimeNumber.py
number = int(input("Enter the number : "))

is_prime = True
for i in range( 2 , number):
  if number % i == 0:
    is_prime = False
    break

if is_prime:
  print(number , "is prime")
else:
  print(number , "is not prime")

File: PrimeNumber.java

class PrimeNumber {
  public static void main(String[] args) {
    int num = 18;

    boolean isPrime = true;
    int i = 2;
    while (i < num) {
      if (num % i == 0) {
        isPrime = false;

      }
      i++;
    }

    if (isPrime) {
      System.out.println("Prime");
    } else {
      System.out.println("not prime");
    }
  }
}